Mathlib/Algebra/Order/GroupWithZero/Unbundled/Basic.lean

```diff
@@ -716,11 +716,14 @@ lemma div_self_le_one (a : G₀) : a / a ≤ 1 := by obtain rfl | ha := eq_or_ne
 end Preorder
 
 section PartialOrder
-variable [PartialOrder G₀] [ZeroLEOneClass G₀] [PosMulReflectLT G₀] {a b c : G₀}
+variable [PartialOrder G₀] [PosMulReflectLT G₀] {a b c : G₀}
 
-@[simp] lemma inv_pos : 0 < a⁻¹ ↔ 0 < a :=
+@[simp] lemma inv_pos : 0 < a⁻¹ ↔ 0 < a := by
   suffices ∀ a : G₀, 0 < a → 0 < a⁻¹ from ⟨fun h ↦ inv_inv a ▸ this _ h, this a⟩
-  fun a ha ↦ flip lt_of_mul_lt_mul_left ha.le <| by simp [ne_of_gt ha, zero_lt_one]
+  intro a ha
+  apply lt_of_mul_lt_mul_left _ ha.le
+  apply lt_of_mul_lt_mul_left _ ha.le
+  simpa [ha.ne']
```

Comment:
Should I add a `ZeroLEOne.of_posMulReflectLT_of_pos` constructor and restore the original proof?

Reply:
nvm, can't prove this constructor (w/o gwz)